respy/solve.py

```diff
@@ -35,7 +37,7 @@ def get_solve_func(params, options):
     """
     optim_paras, options = process_params_and_options(params, options)
 
-    state_space = create_state_space_class(optim_paras, options)
+    state_space = create_state_space_class(options, optim_paras)
```

Comment:
turn the arguments around. it is almost always optim_paras first and options second.